atlasdb-client/build.gradle

```diff
@@ -15,8 +15,17 @@ schemas = [
 
 libsDirName = file('build/artifacts')
 
+task myTestJar(type: Jar) {
+    from sourceSets.test.output
+    classifier = 'tests'
+}
+
 configurations {
-    testArtifacts.extendsFrom testRuntime
+    testArtifacts
+}
+
+artifacts {
+    testArtifacts myTestJar
```

Comment:
where was this artifact being used. Doesnt this mean this will now not be available at testRuntime as we removed it above.